litmus-portal/frontend/package.json

```diff
@@ -38,7 +38,7 @@
     "react-plotly.js": "^2.4.0",
     "react-redux": "^7.1.3",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "^3.4.1",
+    "react-scripts": "^3.4.3",
```

Comment:
Changing react-script version from 3.4.1 to 3.4.3 would cause conflicts in package-lock too. Although 3.4.3 is the latest one I think 3.4.1 was good enough to support a lot. Is there any specific reason for upgrading this?

Reply:
I didn't upgrade this, while npm install it automatically updates @S-ayanide do i fix it to 3.4.1 by removing the ^? cc: @arkajyotiMukherjee @gdsoumya @rajdas98 @chandankumar4